### ISSUE TITLE
Loop Hero doesn't start, missing libraries

### ISSUE THREAD

**Issue by Raumis** [flathub/com.valvesoftware.Steam#728]
**Game information**
Loop Hero
https://store.steampowered.com/app/1282730/Loop_Hero/

**Distribution name and version where applicable**
Oracle Linux Server release 8.3

**Flatpak info**
$ flatpak --version; flatpak --gl-drivers
Flatpak 1.6.2
default
host

**Problem description**
Game does not start.  I ran Steam via `flatpack run com.valvesoftware.Steam`, the logs show 2 libraries are missing.
libnghttp2.so.14
librtmp.so.1

I manually downloaded the libraries and placed them in:
`/home/user/.var/app/com.valvesoftware.Steam/data/Steam/steamapps/common/Loop Hero/assets/linuxlibs`
The game did start after that.

[steam_loop_hero.log](https://github.com/flathub/com.valvesoftware.Steam/files/6094650/steam_loop_hero.log)

**Does this issue reproduce with native Steam**
No, unable

**Comment by nanonyme** [comment]
Yes, this is a bug in the game. It should be bundling these libraries since they're not present in Steam Runtime.

**Comment by matthew-cline** [comment]
Same problem with Flatpak 1.10.2 under Fedora 33.

[Here](https://github.com/flathub/com.valvesoftware.Steam/files/6215381/libs.txt) is the list of native library files the game links when running natively, as taken from `/proc/<PID>/maps`.

**Comment by Mershl** [comment]
Same problem with Flatpak 1.10.2 under Fedora 34.

The game starts up nicely on native Steam.

**Comment by mattiasb** [comment]
Like @nanonyme says this is a bug in the game. They've probably tested on some particular distribution that just so happened to come with some particular dependencies. Then went "Works for me! 🤷" and shipped it.
